### ISSUE TITLE
Lineheight is not respected when generated in linux

### ISSUE THREAD

**Issue by koo9** [QuestPDF/QuestPDF#243]
**Describe the bug**
Lineheight is not respected when running in linux

**To Reproduce**
Compare the generated pdf in windows and linux.

**Expected behavior**
Lineheight should be the same when generated in linux

**Screenshots**
https://1drv.ms/u/s!Atfkc5kdKg4AokWlPcMTN_CVliZs?e=c3dw8g

**Additional context**
running the app in kubernetes container in linux

**Comment by kyugoa** [comment]
Turned out the lineheight ratio is treated differently in linux and windows. if LineHeight = 1.2f in widows, it's approximately 1.4f in linux. so the pdf generated in linux does not look the same as that in windows. (great library for genrating pdf BTW).

**Comment by kyugoa** [comment]
that make sense. will find out if there is an existing bug logged on Skiasharp. thx

**Comment by kyugoa** [comment]
do you mean this issue?  https://github.com/mono/SkiaSharp/issues/838 looks like it's logged a few years back but it's still not addressed.

**Comment by ingridlarssen** [comment]
Did you find a solution to this problem? Having the same problem with mac vs Linux @kyugoa

**Comment by kyugoa** [comment]
no, looks like the bug is not addressed, what you can do is to adjust the line height with a ratio for Linux (by trial and error)

**Comment by MarcinZiabek** [comment]
The text rendering engine has been redesigned in the 2024.3 release. Its capabilities are significantly higher and cover the vast majority of use cases, including the FontFallback feature. Closing this issue.

Since there has been no recent activity, I’m closing this issue. If you experience a similar problem or have a related request, feel free to open a new GitHub issue. I appreciate your feedback! ❤️
